src/Exception/MissingHelperException.php

```diff
@@ -8,10 +8,10 @@
 namespace Zend\Expressive\ZendView\Exception;
 
 use DomainException;
-use Interop\Container\Exception\ContainerException;
+use Psr\Container\ContainerExceptionInterface;
 
 class MissingHelperException extends DomainException implements
-    ContainerException,
+    ContainerExceptionInterface,
```

Comment:
Isn't it here BC Break? If somebody catch exception on `ContainerException` it will stop work.

Reply:
If they were catching `Interop\Container\Exception\ContainerException`, however, and the container in use _implements PSR-11 only_, then there's breakage.